rosa_tutorials/rosa-mobb-deploying-rosa-sts-mode.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-deploying-rosa-sts-mode"]
+= Tutorial: Deploying ROSA in STS mode
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-deploying-rosa-sts-mode
+
+toc::[]
+
+// Mobb content metadata
+// Brought into ROSA product docs 2023-09-21
+// ---
+// date: '2022-05-27'
+// title: Deploying ROSA in STS mode
+// tags: ["AWS", "ROSA", "STS"]
+// weight: 1
+// aliases: ["/docs/quickstart-rosa.md"]
+//authors:
+//  - Paul Czarkowski
+//  - Michael Ducy
+// ---
+
+
+* https://youtu.be/R1T0yk9l6Ys[Quick Introduction by Ryan Niksch (AWS) and Shaozen Ding (Red Hat)].
+
+
+Secure Token Service allows you to deploy ROSA without needing a ROSA admin account. STS uses roles and policies with Amazon STS (secure token service) to gain access to the AWS resources needed to install and operate the cluster.
+
+== Prerequisites
+* xref:../rosa_tutorials/rosa-mobb-prerequisites-tutorial.adoc#rosa-mobb-prerequisites-tutorial[If this is your first time deploying ROSA, follow these prerequisite instructions].
+
+== Deploying a ROSA cluster
+
+. Set environment variables.
++
+[source,terminal]
+----
+   $ export ROSA_CLUSTER_NAME=mycluster
+    export AWS_ACCOUNT_ID=`aws sts get-caller-identity \
+      --query Account --output text`
+    export REGION=us-east-2
+    export AWS_PAGER=""
+----
+. Make sure your ROSA CLI version is correct (v1.2.25 or above).
++
+[source,terminal]
+----
+   $ rosa version
+----
+. Run the ROSA CLI to create your cluster.
+
++
+[source,terminal]
+----
+   $ rosa create cluster --sts --cluster-name ${ROSA_CLUSTER_NAME} \
+      --region ${REGION} --mode auto --yes
```

Comment:
If no flags of account-roles arns are set, the `rosa create cluster ...` command will use the first default bundle of account-roles under the aws account. It's better to set them with specific ones with flags '--role-arn', '--support-role-arn' ,'-worker-iam-role' and '--controlplane-iam-role'